tests/scripts/docker/run_tests.sh

```diff
@@ -6,7 +6,6 @@ docker_pip_cache="/tmp/cache/pip"
 
 cd tests
 
-docker network create run_tests || true
```

Comment:
Is this network not needed any more? I've not been able to find where this network creation happens

Reply:
I had a first draft where I was bootstrapping `localstack` infra by code in the test, I needed for that to use the same network when running the test in docker (localstack spin up containers in the background).
this is not needed anymore since now I spin up the localstack container (I found out that localstack infra by code is not supported and deprecated) and expose the port on the host.
same for elasticsearch container.